app/build.gradle.kts

```diff
@@ -171,11 +171,10 @@ dependencies {
     testImplementation("io.insert-koin:koin-test:$koinVersion")
     testImplementation("io.insert-koin:koin-test-junit4:$koinVersion")
 
-    testImplementation("org.assertj:assertj-core:3.22.0")
+    testImplementation("com.willowtreeapps.assertk:assertk-jvm:0.25")
```

Comment:
AssertJ erases generics when using assertThat, making assertions very bad. 
assertK has the same API